homeassistant/components/apple_tv/services.yaml

```diff
@@ -1,5 +0,0 @@
-apple_tv_authenticate:
```

Comment:
Delete this file if it is not used.

Reply:
That is what I did?

Reply:
Oh sorry I misinterpreted Github's view 😄